docs/ui-coverage/configuration/attributefilters.mdx

```diff
@@ -135,9 +117,38 @@ Whether or not a matched attribute should be used for element identification and
 </body>
 ```
 
-#### Elements shown in UI
+#### Elements shown in UI Coverage
 
 ```
 [ng-include-me="my-button"]
 :nth-child(2)
 ```
+
+### Ignoring dynamic attributes for accurate grouping
+
+```json
+{
+  "uiCoverage": {
+    "attributeFilters": [
+      {
+        "attribute": "data-id",
+        "value": ".*",
+        "include": false
+      }
+    ]
+  }
+}
+```
+
+#### HTML
+
+```xml
+<button data-id="852143838" data-cy="user">Harper</button>
```

Comment:
this is not the best example because data-id is not "significant" by default, so this would not actually do anything.

a better example would be `id` as the dynamic one and ignoring `\\d+` id's, and the buttons also having `name` which is below `id` in priority but would be used if the `id` is filtered out

Reply:
@pstakoun Can you make an edit to the PR? I'm not quite following every step here in this suggestion.

Reply:
@jennifer-shehane I pushed an edit to this page